drkafka/src/main/java/com/pinterest/doctorkafka/config/DoctorKafkaClusterConfig.java

```diff
@@ -34,6 +34,7 @@
   private static final String SECURITY_PROTOCOL = "security.protocol";
   private static final String NOTIFICATION_EMAIL = "notification.email";
   private static final String NOTIFICATION_PAGER = "notificatino.pager";
+  private static final String ENABLE_LOCALITY_AWARENESS = "locality_awareness.enabled";
```

Comment:
as kafka use the term `rack` in the setting, shall we name setting as `rack_awareness.enabled` ? 

     broker.rack 
     Rack of the broker. This will be used in rack aware replication assignment for fault tolerance. 
     Examples: `RACK1`, `us-east-1d`

https://kafka.apache.org/documentation/